test/components/destination_test.dart

```diff
@@ -115,6 +115,31 @@ main() {
         lat: 26.440011,
       ),
     );
+    Point actualEnd =
+        destination(testStart, testDistanceKm, testBearing, Unit.kilometers);
+    expect(actualEnd.coordinates.lat.toStringAsFixed(6),
```

Comment:
should we configure the float precision somewhere globally thus extract the `.toStringAsFixed(6)`? maybe even globally for all tests?